server/App.ts

```diff
@@ -23,6 +23,14 @@ app.use(Logger.consoleLog);
 
 app.use(express.static(path.resolve(__dirname, '../frontend/html/public')));
 
+const cors = require("cors");
+
+var corsOptions = {
+  origin: "http://localhost:8081"
+};
+
+app.use(cors(corsOptions));
```

Comment:
this needs to be wrapped in something like `if process.env.NODE_ENV == 'dev'` or something like that

Reply:
Actually, this is needed for localhost frontend development to work when interacting with the deployed server as well.